DOC Prepare for docs site (#981)
tractorcow-farm/silverstripe-fluent: docs/en/02_migrating-from-single-language.md

When you run `dev/build?flush` again, this adds the records to the database if the locales table is still empty.

> ## Populating initial localised content for existing pages and dataObjects in your default locale

Reviewer: ```suggestion
## Populating initial localised content for existing pages and `DataObject` models in your default locale
```

Author: Updated

Now your site is broken because nothing has been published and added as translated data in your default locale. You can
either manually localise all DataObjects &amp; Pages manually or use one of the automation options below.

### Automated tools for localisation

#### From the CMS (`SiteTree` only)

Use Silverstripe's
built-in [publishall](https://docs.silverstripe.org/en/developer_guides/debugging/url_variable_tools/#building-and-publishing-urls)
tool to publish all Pages in bulk.
Run `/admin/pages/publishall` in your browser and your site will be fixed again and you can start adding translated
content.

*This method will work with Pages only (not localised DataObjects).*

> #### Commandline or `Queued Jobs` (`SiteTree` and `DataObjects`)

Reviewer: ```suggestion
#### Commandline or queued jobs (`SiteTree` and `DataObjects`)
```
That's not a code reference so shouldn't have backticks.

If the capitalisation is really important, we can add an exclusion for it in the linting rules... I suspect it's not that important.

Author: Updated

The [`InitialPageLocalisationTask`](api:TractorCow\Fluent\Task\InitialPageLocalisationTask) and [`InitialDataObjectLocalisationTask`](api:TractorCow\Fluent\Task\InitialDataObjectLocalisationTask) dev tasks may be used to localise and, optionally,

> publish your [`Versioned`](api:SilverStripe\Versioned\Versioned) data (including Pages) from the commandline or queued as a job (if the Queued Jobs module is installed).

Reviewer: ```suggestion
publish your [versioned data](https://docs.silverstripe.org/en/6/developer_guides/model/versioning/#versioning) (including Pages) from the commandline or queued as a job (if the [Queued Jobs module](https://docs.silverstripe.org/en/optional_features/queuedjobs) is installed).
```
Probably more useful linking to the docs about versioning in this case.
Also adding a link to the queued jobs docs section which gets added as part of this work.

Author: Updated

> publish your [versioned data](https://docs.silverstripe.org/en/6/developer_guides/model/versioning/#versioning) (including Pages) from the commandline or queued as a job (if the [Queued Jobs module](https://docs.silverstripe.org/en/optional_features/queuedjobs) is installed).

Reviewer: ```suggestion
publish your [versioned data](https://docs.silverstripe.org/en/developer_guides/model/versioning/#versioning) (including Pages) from the commandline or queued as a job (if the [Queued Jobs module](https://docs.silverstripe.org/en/optional_features/queuedjobs) is installed).
```
Oops, sorry, my suggestion had the version number in it but it shouldn't be there.

Author: Updated

`InitialPageLocalisationTask` - localise all `SiteTree` objects (Pages)

`InitialDataObjectLocalisationTask` - localise all Fluent-enabled DataObjects (excluding `SiteTree`)

1. Example: Localise all pages (default, without publishing)

   ```bash
   dev/tasks/initial-page-localisation-task
   ```

1. Example: Localise &amp; publish all pages

    ```bash
    dev/tasks/initial-page-localisation-task publish=1
    ```

1. Example: Localising pages in batches can be done by using the `limit` option.
   This will localise &amp; publish five pages on each run.

    ```bash
    dev/tasks/initial-page-localisation-task publish=1&limit=5
    ```

> 1. Example: All the same functionality is available for localising all `DataObject`s, including `Versioned` and non-Versioned classes

Reviewer: ```suggestion
1. Example: All the same functionality is available for localising all `DataObject` models, including versioned and non-versioned classes
```
or "subclasses"

Also makes more sense to just un-capitalise "Versioned" here since `non-Versioned` doesn't really make sense with a capital.

Author: Updated
